Changing CUDA cxx compiler VERSION_LESS to 12.4

franky.backeljauw@uantwerpen.be
michele.pugno@uantwerpen.be
--- amber24_src/cmake/CudaConfig.cmake.orig	2024-04-29 19:47:47.000000000 +0200
+++ amber24_src/cmake/CudaConfig.cmake	2025-05-08 16:19:24.440518551 +0200
@@ -84,7 +84,7 @@
 		       ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 13.3
 			AND CUDA_VERSION VERSION_GREATER_EQUAL 12.4
 			AND CUDA_VERSION VERSION_LESS_EQUAL 12.4 )
-		    OR ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.3
+		    OR ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.4
 			AND CUDA_VERSION VERSION_GREATER_EQUAL 12.1
 			AND CUDA_VERSION VERSION_LESS_EQUAL 12.3 )
 		    OR ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.2

--- amber24_src/AmberTools/src/quick/cmake/CudaConfig.cmake.orig	2024-04-29 19:47:47.000000000 +0200
+++ amber24_src/AmberTools/src/quick/cmake/CudaConfig.cmake	2025-05-08 16:35:01.753472296 +0200
@@ -84,7 +84,7 @@
 		       ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 13.3
 			AND CUDA_VERSION VERSION_GREATER_EQUAL 12.4
 			AND CUDA_VERSION VERSION_LESS_EQUAL 12.4 )
-		    OR ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.3
+		    OR ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.4
 			AND CUDA_VERSION VERSION_GREATER_EQUAL 12.1
 			AND CUDA_VERSION VERSION_LESS_EQUAL 12.3 )
 		    OR ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.2

--- amber24_src/AmberTools/src/quick/quick-cmake/QUICKCudaConfig.cmake.orig	2024-04-29 19:47:47.000000000 +0200
+++ amber24_src/AmberTools/src/quick/quick-cmake/QUICKCudaConfig.cmake	2025-05-08 16:35:27.776943777 +0200
@@ -190,7 +190,7 @@
             ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 13.3
               AND CUDA_VERSION VERSION_GREATER_EQUAL 12.4
               AND CUDA_VERSION VERSION_LESS_EQUAL 12.4 )
-        OR ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.3
+        OR ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.4
               AND CUDA_VERSION VERSION_GREATER_EQUAL 12.1
               AND CUDA_VERSION VERSION_LESS_EQUAL 12.3 )
         OR ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.2
